db/practicalplants.js

```diff
@@ -105,6 +105,19 @@ function readCrops() {
       replaceValue(object, property, ['Yes', 'True'], 'true');
     });
 
+    for (const property of NAME_PROPERTIES) {
+      if (object[property]) {
+        if (/[a-z]/.test(object[property][0])) {
```

Comment:
It is probably safer to use `letter === letter.toLowerCase()` for possible non-ASCII letters.

Reply:
Even better `character !== character.toUpperCase()` for possible other characters.

Reply:
Never mind, it is okay to limit the replace to only `/[a-z]/`. In the test it was already checked that these properties always start with `/[a-zA-Z]/`.